### DISCUSSION TITLE
Different speed gameplay

### DISCUSSION THREAD

**Discussion by jvondermarck** [AlmasB/FXGL#1161] [Q&A] [answered]
Hi @AlmasB, a friend and I when we run the FXGL game, we have a different speed gameplay. It looks faster on my computer but on my friend the game looks running slower. 

Gameplay of my friend : https://www.youtube.com/shorts/ns6axLTgJyI 
My gameplay : https://www.youtube.com/watch?v=9Y8NQGIS4X8&feature=youtu.be 

We both have a really good computer so that's weird.
Our GitHub project : https://github.com/dylan-power/dinosaur-exploder

**Comment by AlmasB** [comment, marked as answer]
Hi, you can enable profiling via `GameSettings` object to see whether you have different results.

By default, FXGL relies on JavaFX to lock the frame rate, which in turn means the loop runs at the display refresh rate speed. So, for example, if you have a 120fps display, the game will run twice as fast. To change this behavior, you can lock the fps to 60, again via `GameSettings`.

**Reply by jvondermarck** [reply]
Oh thank you for your answer, we're gonna lock the fps to 60. 👌

**Reply by jvondermarck** [reply]
@AlmasB Sorry but I didn't found in the `GameSettings` class how to lock the fps to 60 ? Do you have any idea how to do it ?

**Reply by AlmasB** [reply]
It should be: `settings.setTicksPerSecond(60)` inside the `initSettings()` callback.
